tests/mmptest/src/MMPTest.cs

```diff
@@ -240,6 +240,23 @@ public void Unified_HelloWorld_ShouldHaveNoWarnings ()
 			});
 		}
 
+        [Test]
+        public void Unified_HelloWorld_ShouldWarnOn32Bit ()
+        {
+            RunMMPTest (tmpDir => {
+                TI.UnifiedTestConfig test = new TI.UnifiedTestConfig (tmpDir) {
+                    CSProjConfig = "<XamMacArch>i386</XamMacArch>"
+                };
+
+                string buildResults = TI.TestUnifiedExecutable (test).BuildOutput;
+                Assert.True (buildResults.Contains ("MM0134"), "32-bit Modern did not contain MM0134");
+
+                test.XM45 = true;
+                buildResults = TI.TestUnifiedExecutable (test).BuildOutput;
+                Assert.True (buildResults.Contains ("MM0134"), "32-bit Full did not contain MM0134");
+            });
+        }
```

Comment:
indent seems wrong

Reply:
New machine, i thought i hammered VSfM formatting into correct settings but apparently now.

We don't have them somewhere I can apply directly, do we?